app/controllers/api/internal/v2/laa_references_controller.rb

```diff
@@ -37,6 +39,17 @@ def enforce_contract!(contract)
           end
         end
 
+        def check_defendant_presence!
+          defendant_id = transformed_params[:defendant_id]
+          laa_reference = LaaReference.find_by(defendant_id: defendant_id)
+
+          if laa_reference.nil?
+            raise ActiveRecord::RecordNotFound, "Defendant NOT FOUND: defendant_id:'#{defendant_id}'"
+          elsif laa_reference.linked == false
+            raise ActiveRecord::RecordNotFound, "Defendant is already non-linked: defendant_id:'#{defendant_id}'"
```

Comment:
instead of `non-linked` could `unlinked` be used in the message ? Only so that it follows the current standard.

Reply:
Yes, probably it's better to use "unlinked".